ci/citest/acceptance-test/tests/cmd_template_test.go

```diff
@@ -0,0 +1,27 @@
+// +build acceptance
+
+package tests
+
+import (
+	"testing"
+)
+
+//go:generate go-bindata -pkg tests -o fixtures.bindata.go ./fixtures
+
+func init() {
+	if err := RestoreAsset("/var/tmp", "fixtures/lxc.json"); err != nil {
+		panic(err)
+	}
+}
+
+func TestCmdTemplateValidate(t *testing.T) {
+	RunCmdAndReportFail(t, "openvdc", "template", "validate", "centos/7/lxc")
+	RunCmdAndReportFail(t, "openvdc", "template", "validate", "/var/tmp/fixtures/lxc.json")
+	RunCmdAndReportFail(t, "openvdc", "template", "validate", "https://raw.githubusercontent.com/axsh/openvdc/master/templates/centos/7/lxc.json")
```

Comment:
Is it enough to only check the exit status here? If it is, maybe also include some negative tests to confirm that the validation command will indeed fail when called on an invalid template.